crates/ruff/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -558,19 +564,31 @@ fn check_duplicates(checker: &mut Checker, values: &Expr) {
     else {
         return;
     };
+    if elts.len() < 2 {
+        return;
+    }
 
     let mut seen: FxHashMap<ComparableExpr, usize> =
         FxHashMap::with_capacity_and_hasher(elts.len(), BuildHasherDefault::default());
+    let mut prev = &elts[0];
```

Comment:
I think this should either be optional _or_ we should insert it into `seen` outside of the loop and skip the first element.